charts/datree-admission-webhook/templates/_helpers.tpl

```diff
@@ -23,3 +23,10 @@ helm.sh/chart: {{ template "datree.chart" . }}
 {{- define "datree.namespace" -}}
 {{- default .Release.Namespace .Values.namespace  -}}
 {{- end -}}
+
+{{/* The imagePullSecret */}
+{{- define "imagePullSecret" }}
+{{- with .Values.image.imageCredentials }}
+{{- printf "{\"auths\":{\"%s\":{\"username\":\"%s\",\"password\":\"%s\",\"email\":\"%s\",\"auth\":\"%s\"}}}" .registry .username .password .email (printf "%s:%s" .username .password | b64enc) | b64enc }}
+{{- end }}
+{{- end }}
```

Comment:
When installing the helm I get 
```
Error: INSTALLATION FAILED: parse error at (datree-admission-webhook/templates/_helpers.tpl:27): comment ends before closing delimiter
helm.go:84: [debug] parse error at (datree-admission-webhook/templates/_helpers.tpl:27): comment ends before closing delimiter
INSTALLATION FAILED
main.newInstallCmd.func2
	helm.sh/helm/v3/cmd/helm/install.go:127
github.com/spf13/cobra.(*Command).execute
	github.com/spf13/cobra@v1.3.0/command.go:856
github.com/spf13/cobra.(*Command).ExecuteC
	github.com/spf13/cobra@v1.3.0/command.go:974
github.com/spf13/cobra.(*Command).Execute
	github.com/spf13/cobra@v1.3.0/command.go:902
main.main
	helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
	runtime/proc.go:255
runtime.goexit
	runtime/asm_arm64.s:1133
make: *** [helm-install-staging] Error 1
```

In addition `.Values.image.imageCredentials` is incorrect, `imageCredentials` is not inside `image`

Reply:
Sorry for the inconvenience. I fixed that :)
I think now it should be installed fine 👍